packages/hint-compat-api/src/compat-api-css-next.ts

```diff
@@ -0,0 +1,162 @@
+/**
+ * @fileoverview Hint to validate if the HTML, CSS and JS APIs of the project are deprecated or not broadly supported
+ */
+
+import { Category } from 'hint/dist/src/lib/enums/category';
+import { HintScope } from 'hint/dist/src/lib/enums/hintscope';
+import { HintContext } from 'hint/dist/src/lib/hint-context';
+import { IHint, HintMetadata } from 'hint/dist/src/lib/types';
+import { debug as d } from 'hint/dist/src/lib/utils/debug';
+import { StyleParse } from '@hint/parser-css/dist/src/types';
+import { forEach } from 'lodash';
+import { CompatApi, userBrowsers, CompatCSS } from './helpers';
+import { MDNTreeFilteredByBrowsers, BrowserSupportCollection } from './types';
+import { SupportBlock } from './types-mdn.temp';
+import { browserVersions } from './helpers/normalize-version';
+
+const debug: debug.IDebugger = d(__filename);
+
+/*
+ * ------------------------------------------------------------------------------
+ * Public
+ * ------------------------------------------------------------------------------
+ */
+
+export default class implements IHint {
+
+    public static readonly meta: HintMetadata = {
```

Comment:
Just a heads-up: we're changing how `HintMetadata` is declared to facilitate loading it separate from hints (e.g. for building documentation). See `hint-typescript-config` in PR #1480 for how I'm proposing we do this for multi-hints.

Reply:
Ok thanks